src/nunjucks.js

```diff
@@ -1,14 +1,7 @@
 import chroma from 'chroma-js'
-import { Swig } from 'swig'
-import swigExtras from 'swig-extras'
-import swigFilters from 'swig/lib/filters'
+import nunjucks from 'nunjucks'
 
-const swig = new Swig()
-export default swig
-
-swigExtras.useFilter(swig, 'split')
-swigExtras.useFilter(swig, 'trim')
-swigExtras.useFilter(swig, 'groupby')
+let nunjucksEnv = nunjucks.configure('views')
```

Comment:
It should be straighten to `nunjucks.configure(path.resolve(__dirname, '..', 'views'))`.